components/navigation/navigation-styles.tsx

```diff
@@ -19,19 +19,22 @@ export const NavBar = styled.ul`
 
   ${media.tablet`font-size: 2.2rem;`}
 
-  ${media.phone`
+  ${(props) => media.phone`
     position: fixed;
     background-color: white;
-    left: ${(props) => (props.mobileMenuShown ? '0' : '-30.7rem')};
+    left: -30.7rem;
     top: -5.2rem;
     height: 100%;
     width: 22.7rem;
     box-shadow: 20px 0px 54px -4px rgba(0,0,0,1);
     flex-direction: column;
     z-index: 1000;
-    background: url(${MobileMenuBackground.src}) center top;
+    background: ${css`url(${MobileMenuBackground.src}) center top`};
```

Comment:
Why is this preferred?

Reply:
It wasn't necessary. I added this earlier because due to the type errors in `utilities/media.ts`. I've removed it now.